_pages/install/integrations/cdc-integration.md

```diff
@@ -0,0 +1,247 @@
+    $contentCatalog=electronics-spaContentCatalog
+    $contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+    $jarResourceCms=jar:de.hybris.platform.gigya.gigyaloginaddon.constants.GigyaloginaddonConstants&/gigyaloginaddon/import/cockpit/cmscockpit
+    $lang=en
+
+    # Import modulegen config properties into impex macros
+    UPDATE GenericItem[processor=de.hybris.platform.commerceservices.impex.impl.ConfigPropertyImportProcessor];pk[unique=true]
+    $jarResourceCms=$config-jarResourceCmsValue
+
+    # Create PageTemplates
+
+    INSERT_UPDATE PageTemplate;$contentCV[unique=true];uid[unique=true];name;frontendTemplateName;restrictedPageTypes(code);active[default=true]
+    # Templates without a frontendTemplateName
+    ;;GigyaLoginPageTemplate;Gigya Login Page Template;;ContentPage;false;
+
+
+    # Create Content Slots
+    ##INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];name;active
+    ##;;BodyContent;Body Content Slot;true
+    ##;;BottomContent;Bottom Content Slot;true
+
+
+    # GIGYA Login Page Template
+    INSERT_UPDATE ContentSlotName;name[unique=true];template(uid,$contentCV)[unique=true][default='GigyaLoginPageTemplate'];validComponentTypes(code);compTypeGroup(code)
+    ;SiteLogo;;;logo
+    ;HeaderLinks;;;headerlinks
+    ;SearchBox;;;searchbox
+    ;MiniCart;;;minicart
+    ;NavigationBar;;;navigation
+    ;Footer;;;footer
+    ;TopHeaderSlot;;;wide
+    ;BottomHeaderSlot;;;wide
+    ;BodyContent;;;wide
+    ;BottomContent;;;wide
+    ;PlaceholderContentSlot;;;
+
+
+    INSERT_UPDATE ContentSlotForTemplate;$contentCV[unique=true];uid[unique=true];position[unique=true];pageTemplate(uid,$contentCV)[unique=true][default='GigyaLoginPageTemplate'];contentSlot(uid,$contentCV)[unique=true];allowOverwrite
+    ;;SiteLogo-GigyaLoginPage;SiteLogo;;SiteLogoSlot;true
+    ;;HeaderLinks-GigyaLoginPage;HeaderLinks;;HeaderLinksSlot;true
+    ;;SearchBox-GigyaLoginPage;SearchBox;;SearchBoxSlot;true
+    ;;MiniCart-GigyaLoginPage;MiniCart;;MiniCartSlot;true
+    ;;NavigationBar-GigyaLoginPage;NavigationBar;;NavigationBarSlot;true
+    ;;Footer-GigyaLoginPage;Footer;;FooterSlot;true
+    ;;TopHeaderSlot-GigyaLoginPage;TopHeaderSlot;;TopHeaderSlot;true
+    ;;BottomHeaderSlot-GigyaLoginPage;BottomHeaderSlot;;BottomHeaderSlot;true
+    ##;;BodyContent-GigyaLoginPage;BodyContent;;BodyContent;true
+    ##;;BottomContent-GigyaLoginPage;BottomContent;;BottomContent;true
+    ;;PlaceholderContentSlot-GigyaLoginPage;PlaceholderContentSlot;;PlaceholderContentSlot;true
+    ;;HomepageLink-GigyaLoginPage;HomepageNavLink;;HomepageNavLinkSlot;true
+
+
+    # Functional Content Pages
+    INSERT_UPDATE ContentPage;$contentCV[unique=true];uid[unique=true];name;masterTemplate(uid,$contentCV);label;defaultPage[default='true'];approvalStatus(code)[default='approved'];homepage[default='false'];
```

Comment:
I think we should remove pages for registration,forgot password, change password.